cmd/cliconfig.go

```diff
@@ -61,14 +60,20 @@ func newCliClient(account string) *gsclient {
 		}
 	}
 
-	clientConf := &clientConfig{
-		apiURL:     defaultAPIURL,
-		userUUID:   ac.UserID,
-		userToken:  ac.Token,
-		userAgent:  "gscloud",
-		httpClient: http.DefaultClient,
-	}
-	return newClient(clientConf)
+	// clientConf := &clientConfig{
+	// apiURL:     defaultAPIURL,
+	// userUUID:   ac.UserID,
+	// userToken:  ac.Token,
+	// userAgent:  "gscloud",
+	// httpClient: http.DefaultClient,
+	// }
+	// return newClient(clientConf)
```

Comment:
Do not commit commented out code. Remove it.

Here and other hunks.

Reply:
removed